ACE/ace/config-g++-common.h

```diff
@@ -16,6 +16,13 @@
 
 #define ACE_TEMPLATES_REQUIRE_SOURCE
 
+#define GCC_VERSION (__GNUC__ * 10000 \
```

Comment:
As you add a new macro to ACE please add ACE_ prefix, add some examples so that people know how to use it, and check if there are more places where we can use this macro

Reply:
I removed that define I didn't realize gcc version is already defined right above it